content/manuals/engine/storage/volumes.md

```diff
@@ -584,8 +584,8 @@ Each volume driver may have zero or more configurable options.
 
 ```console
 $ docker run -d \
-  --name sshfs-container \
-  --mount type=volume,volume-driver=vieux/sshfs,src=sshvolume,target=/app,volume-opt=sshcmd=test@node2:/home/test,volume-opt=password=testpassword \
+  --name rclone-container \
+  --mount type=volume,volume-driver=rclone,src=rclonevolume,target=/app,volume-opt=remote=remotehost:home/test,volume-opt=rclone_config=$(cat /path/to/rclone.conf) \
```

Comment:
Same for this one (`remotehost`).

Mostly to be own the safe side and we don't provide an example that could make a connection to some untrusted domain.